src/qt/dc.cpp

```diff
@@ -739,13 +739,13 @@ bool wxQtDCImpl::DoBlit(wxCoord xdest, wxCoord ydest,
 {
     wxQtDCImpl *implSource = (wxQtDCImpl*)source->GetImpl();
 
-    QImage *qtSource = implSource->GetQImage();
+    QPixmap *qtSource = implSource->GetQPixmap();
 
     // Not a CHECK on purpose
     if ( !qtSource )
         return false;
 
-    QImage qtSourceConverted = *qtSource;
+    QImage qtSourceConverted = qtSource->toImage();
```

Comment:
Why exactly is the conversion needed here? What would be wrong with using `qtSource` and [`drawPixmap()`](http://doc.qt.io/qt-5/qpainter.html#drawPixmap-2)?

Reply:
The diff, doesn't show this very well,  but the reset of the function converts the image to remove the mask if useMask is false.  This isn't a change I've made it was how it was previously implemented.

It Might be neater to split the logic and draw the pixmap directly if we are using the mask and convert to an image if now.  

I'm happy to make this change if you want.

Reply:
I wonder if the mask information is really used in other ports beside MSW.
IOW I don't know, but if you are doing that maybe you can test without converting to `QImage` (and probably ignoring the mask information) and see if there are any drawbacks.